src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discont36 = 20;
+  const discont7 = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const bigDiscount = 50;
```
What is 36? Don't hardcore variable's name for some value. It may change in future